### PR TITLE
feat: add additional theme option

### PR DESCRIPTION
**Pull request by braven112**
# Alaska Airlines Pull Request

**Fixes:** # (issue, if applicable)

## Summary:

Updates the names of the themes that are available and removes old themes

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/apiExamples.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
 <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../api.md) -->
 <!-- The below content is automatically added from ./../api.md -->
 
-# auro-themeswitcher
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>
```
Comment (jason-capsule42) [inline demo/apiExamples.md:4]: Are these necessary? Didn't seem like it when I ran the localhost.

File: demo/apiExamples.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 | Property | Attribute | Type    | Default                                          | Description                                      |
 |----------|-----------|---------|--------------------------------------------------|--------------------------------------------------|
-| [themes](#themes) | `themes`  | `Array` | [{"label":"Auro 4.x","url":"https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css"},{"label":"Auro 5.x","url":"https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css"}] | This accepts an array of JSON object outlining the themes to support. |
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>| [themes](#themes) | `themes`  | `Array` | [{"label":"Auro 4.x","url":"https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css"},{"label":"Auro 5.x","url":"https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css"}] | This accepts an array of JSON object outlining the themes to support. |
```
Comment (jason-capsule42) [inline demo/apiExamples.md:14]: Same here

File: demo/apiExamples.md [hunk 3]
```diff
@@ -24,19 +32,30 @@
...
-</auro-accordion>
 
-### Property Examples
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>```
+
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a><!-- AURO-GENERATED-CONTENT:END -->
+
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a></auro-accordion>
+
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>
```
Comment (jason-capsule42) [inline demo/apiExamples.md:50]: There are so many back to top links!

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
See comments, also a build needs to get run in order to generate the documentation correctly.

File: demo/apiExamples.md (hunk 1 above)
Comment (braven112) [reply demo/apiExamples.md:4]: I don't think so these were added when I ran a build.  I can revert that part of the commit.

**Comment by braven112** [comment]
> See comments, also a build needs to get run in order to generate the documentation correctly.

The weird doc updates were when I ran `npm run build`.  I've reverted that commit.  I'm not sure how to generate the code other than with that command.  Do you mind checking on your end and let me know what we need for doc updates.

**Comment by blackfalcon** [comment]
If you update this set of code to have all the generated docs, then a local build is not required with a document change. 
https://github.com/AlaskaAirlines/auro-themeswitcher/blob/619da36fda5e763daae269622df9ffe8b9509ad0/package.json#L114-L118

This will happen in the remote 
https://github.com/AlaskaAirlines/auro-themeswitcher/blob/619da36fda5e763daae269622df9ffe8b9509ad0/.github/workflows/testPublish.yml#L41-L43

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**